src/api/validators.ts

```diff
@@ -13,3 +13,17 @@ export const getCharacterSchema = Joi.object<GetCharacterRequestParams>({
     .valid(...Realms)
     .required(),
 }).required();
+
+export interface getMatchesParams {
+  name: CharacterName;
+  realm: Realm;
+  continuation_token: string;
+}
+
+export const getMatchesSchema = Joi.object<getMatchesParams>({
+  name: Joi.string().valid().required(),
```

Comment:
Would it be a good measure to figure out the constraints that are put on players of Warmane's servers when they create a username? It could be as easy as going to create a WoW account, where they'd hopefully have a list that looks something like "Username must be between 5-16 characters, no punctuation, etc."
On the other hand, if it was a blind list or the rules have changed at all over the past few decades, it would be a bit more difficult.
I suppose we could just implement a few basic rules to close up any obvious vulnerabilities, though Joi might do that already.

Reply:
Yeah it might be better to exclude whitespace, numbers or punctuation. It's not known exactly what the rules look like on the WoW side of things. I know they support unicode to an extent. I've seen some people in game running around with a bunch of Korean characters in their name.

Reply:
I'm not particularly concerned with vulnerabilities because we don't really have any threat actors targeting us, but it would be good to know that they can't inject Dynamo commands.

Reply:
I don't know man, word on the street is that The Dark Army has us in their sights...